components/shapediver/parameter/select/SelectCarouselComponent.tsx

```diff
@@ -0,0 +1,202 @@
+import Icon from "@AppBuilderShared/components/ui/Icon";
+import TextWeighted from "@AppBuilderShared/components/ui/TextWeighted";
+import {IconTypeEnum} from "@AppBuilderShared/types/shapediver/icons";
+import {Carousel} from "@mantine/carousel";
+import "@mantine/carousel/styles.css";
+import {
+	Card,
+	Image,
+	MantineThemeComponent,
+	Stack,
+	Text,
+	useProps,
+} from "@mantine/core";
+import React, {useCallback, useMemo} from "react";
+import classes from "./SelectCarouselComponent.module.css";
+import {
+	SelectCardStyleProps,
+	SelectCarouselStyleProps,
+	SelectComponentProps,
+	SelectImageStyleProps,
+	SelectStackStyleProps,
+	SelectTextStyleProps,
+	SelectTextWeightedStyleProps,
+} from "./SelectComponent";
+
+interface StyleProps {
+	carouselProps: SelectCarouselStyleProps;
+	cardProps: SelectCardStyleProps;
+	imageProps: SelectImageStyleProps;
+	stackProps: SelectStackStyleProps;
+	labelProps: SelectTextWeightedStyleProps;
+	descriptionProps: SelectTextStyleProps;
+	showLabel: boolean;
+}
+
+export const defaultStyleProps: Partial<StyleProps> = {
+	carouselProps: {
+		align: "start",
+		height: "auto",
+		loop: true,
+		slideGap: {base: "xs"},
+		slideSize: {
+			base: "100%",
+			"16em": "50%",
+			"32em": "33.333333%",
+			"48em": "25%",
+			"64em": "20%",
+		},
+		type: "container",
+	},
```

Comment:
@LemToUp here we can set `withIndicators: false`, that solves what Mathieu asked for